tests/fixedPrice.test.js

```diff
@@ -3982,6 +4308,589 @@ describe('Wrapped ZIL', () => {
+
+
+    //Call with contract owner, this should be good
+    tx = await enableSignedOrder(
+      fixedPriceContract,
+      accounts.verifier.privateKey,
+      '0x' + getPubKeyFromPrivateKey(accounts.verifier.privateKey)
+    )
+    expect(tx.receipt.success).toEqual(true)
+
+
+    //Enable again, this should fail
+    tx = await enableSignedOrder(
+      fixedPriceContract,
+      accounts.verifier.privateKey,
+      '0x' + getPubKeyFromPrivateKey(accounts.verifier.privateKey)
+    )
+
+    console.log(tx.receipt.exceptions)
+    expect(tx.receipt.success).toEqual(false)
+    expect(tx.receipt.exceptions).toEqual([
+      {
+        line: 1,
+        message: 'Exception thrown: (Message [(_exception : (String "Error")) ; (code : (Int32 -27))])'
+      },
+      { line: 1, message: 'Raised from RequireContractOwner' },
+      { line: 1, message: 'Raised from EnableSignedOrder' }
+    ])  
+
+
+    //Call with non contract owner, this should fail since it's already enabled
+    tx = await disableSignedOrder(
+      fixedPriceContract,
+      accounts.forbidden.privateKey,
+    )
+
+    expect(tx.receipt.success).toEqual(false)
+    expect(tx.receipt.exceptions).toEqual([
+      {
+        line: 1,
+        message: 'Exception thrown: (Message [(_exception : (String "Error")) ; (code : (Int32 -1))])'
+      },
+      { line: 1, message: 'Raised from DisableSignedOrder' }
+    ])    
+
+
+    //Disable signed order mode, this should be good
+    tx = await disableSignedOrder(
+      fixedPriceContract,
+      accounts.verifier.privateKey
+    )
+    expect(tx.receipt.success).toEqual(true)    
+
+    //Disable again, this should fail since it's already disabled
+    tx = await disableSignedOrder(
+      fixedPriceContract,
+      accounts.verifier.privateKey,
+    )
+
+    console.log(tx.receipt.exceptions)
+    expect(tx.receipt.success).toEqual(false)
+    expect(tx.receipt.exceptions).toEqual([
+      {
+        line: 1,
+        message: 'Exception thrown: (Message [(_exception : (String "Error")) ; (code : (Int32 -26))])'
+      },
+      { line: 1, message: 'Raised from RequireContractOwner' },
+      { line: 1, message: 'Raised from DisableSignedOrder' }
+    ])  
+
+  })
+
+  test('FulfillOrderSigned: Successful case, sell side', async () => {
+    const fixedPriceContract = await zilliqa.contracts.at(fixedPriceAddress)
+
+    //enable and set verifier key
+    tx = await enableSignedOrder(
+      fixedPriceContract,
+      accounts.verifier.privateKey,
+      '0x' + getPubKeyFromPrivateKey(accounts.verifier.privateKey)
+    )
+    expect(tx.receipt.success).toEqual(true)
+        
+    tx = await fulfillOrderSigned(
+      accounts.verifier.privateKey,
+      accounts.nftBuyer.privateKey,
+      fixedPriceAddress,
+      nftTokenAddress.toLowerCase(),
+      tokenId,
+      side,
+      salePrice,
+      zero_address,
+      accounts.nftBuyer.address
+    )
+    console.log(tx.receipt)
+    expect(tx.receipt.success).toEqual(true)
```

Comment:
@inamirandaz Can also add more validation like

- correct event generated
- match parameters with event data
- match balance of user/buyer/contract (just to make sure that user got what he/she supported to get)